frontend/public/locales/fr/translation.json

```diff
@@ -0,0 +1,303 @@
+    "extensionNotAvailableOnYourBrowser": "L'extension n'est malheureusement pas disponible pour votre navigateur web. Elle est compatible avec <2>Mozilla Firefox</2>, et <4>Google Chrome</4> ou <6>Chromium</6>.",
+    "collaborativeContentRecommendations": "Recommandations collaboratives de contenus",
+    "whatIsTournesol": "Qu'est-ce que Tournesol ?",
+    "tournesolPlatformDescription": "Tournesol est une plateforme <1>libre et open source</1> dont le but est d'identifier les vidéos d'utilité publique de grande qualité, de <3>manière collaborative</3>, en s'appuyant sur le jugement de contributeurs. Nous espérons rendre les algorithmes d'aujourd'hui et de demain <6>robustement bénéfique</6> à grande échelle, pour toute l'humanité."
+  },
+  "reset": {
+    "ifUserExistsResetLinkWillBeSent": "C'est fait !<1></1>Si l'identifiant existe, un courriel avec un lien de réinitialisation sera envoyé.",
+    "resetYourPassword": "Réinitialisez votre mot de passe",
+    "failToSendResetLink": "L'envoi du courriel a échoué.",
+    "sendResetEmailButton": "Envoyer un courriel de réinitialisation",
+    "backToLogIn": "Retourner à la page de connexion",
+    "passwordModifiedSuccessfully": "Mot de passe bien modifié. Vous pouvez maintenant vous connecter.",
+    "newPassword": "Nouveau mot de passe",
+    "confirmNewPassword": "Confirmez votre nouveau mot de passe",
+    "passwordsDoNotMatch": "Les mots de passe ne correspondent pas",
+    "resetPasswordButton": "Réinitialiser le mot de passe"
+  },
+  "myRateLaterListPage": {
+    "title": "À comparer plus tard"
+  },
+  "signup": {
+    "successMessage": "Bravo !<1></1>Un lien de vérification vient d'être envoyé à <3>{{email}}</3> .",
+    "title": "Inscription",
+    "accountCreationFailed": "Création du compte échouée.",
+    "iAgreeWithThePrivacyPolicy": "J'ai lu et j'accepte la <2>politique de confidentialité</2>"
+  },
+  "emailAddress": "Adresse e-mail",
+  "confirmYourPassword": "Confirmez votre mot de passe",
+  "signUpButton": "S'inscrire",
+  "verify": {
+    "verifyRegistration": "Vérifier l'inscription",
+    "yourAccountIsNowVerified": "Votre compte est maintenant vérifié.",
+    "yourNewEmailAddressIsNowVerified": "Votre nouvelle adresse e-mail est maintenant vérifiée.",
+    "verificationFailed": "La vérification a échoué"
+  },
+  "myRatedVideosPage": {
+    "title": "Mes vidéos comparées"
+  },
+  "recommendationsPage": {
+    "title": "Recommandations"
+  },
+  "noVideoCorrespondsToSearchCriterias": "Aucune vidéo ne correspond à vos critètes de recherche.",
+  "actions": {
+    "compareNow": "Comparer !",
+    "videoAddedToRateLaterList": "Vidéo bien ajoutée à votre liste de vidéos à comparer.",
+    "videoAlreadyInRateLaterList": "La vidéo se trouve déjà dans votre liste.",
+    "rateLater": "Comparer plus tard",
+    "remove": "Supprimer",
+    "videoDeletedFromRateLaterList": "Vidéo bien supprimée de votre liste de vidéos à comparer."
```

Comment:
The rate later list is sometimes referred as "vidéos à comparer" or "À comparer plus tard". To make consistent?